controllers/feeds/deleteAnonymousPost.js

```diff
@@ -26,11 +26,12 @@ module.exports = async (req, res) => {
   }
 
   try {
-    const getstreamDeleteResult = await getstreamService.deleteFeedById(
-      'user_anon',
-      postId,
-      anonymousUser?.user_id
-    );
+    const getstreamDeleteResult = await getstreamService.deleteFeedById({
+      feedGroup: 'user_anon',
+      activityId: postId,
+      userId: anonymousUser?.user_id,
+      token
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes look good, but there's a potential issue. If `anonymousUser` is null or undefined, `anonymousUser?.user_id` will be undefined and passed as `userId` to `deleteFeedById`. This could lead to unexpected behavior if the function doesn't handle this case properly. Consider adding a check for `anonymousUser` before this call.
```diff
+    if (!anonymousUser) {
+      throw new Error('Anonymous user not found');
+    }
     const getstreamDeleteResult = await getstreamService.deleteFeedById({
       feedGroup: 'user_anon',
       activityId: postId,
       userId: anonymousUser?.user_id,
       token
     });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->